packages/wallets/src/components/WalletDepositCryptoAddress/WalletDepositCryptoAddress.tsx

```diff
@@ -26,14 +26,14 @@ const WalletDepositCryptoAddress = () => {
 
     return (
         <div className='wallets-deposit-crypto-address'>
-            <QRCode value={deposit_crypto_address || ''} size={is_mobile ? 128 : 160} className='' />
+            <QRCode className='' size={is_mobile ? 128 : 160} value={deposit_crypto_address || ''} />
```

Comment:
is the `className=''` required here? 👀

Reply:
I believe not 🤔 But I'm not touching files, This is ESLint auto-fix sorting props.

Although I will remove it for now 🙇🏻